back-end-side/Controllers/Sorting.cs

```diff
@@ -29,7 +31,7 @@ public Sorting(ExpensesContext context)
         public readonly string[] CarMaintenanceShops = { "CIRCLE K", "VIADA", "BOLT", "STOVA", "RIDE SHARE", "SUSISIEKIMO PASLAUGOS", "CITYBEE" };
         public readonly string[] HouseMaintenanceShops = { "JYSK", "MOKI VEZI", "SENUKAI", "CONSILIUM OPTIMUM", "TELE2", "telia", "TOPO CENTRAS", "BITĖ", "GO3", "IKEA" };
         public readonly string[] EntertainmentShops = { "CYBERX", "SPOTIFY", "BASKETNEWS", "OCULUS DIGITAL", "STEAM", "ŽALGIRIO KREPŠINIO CENTRAS" };
-        public SortingModel SortToCategories()
+        public SortingModel SortToCategories(string pickedDate = "aa")
```

Comment:
Why is pickedDate by default set to "aa"?

Reply:
Left by mistake, changed it